use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -192,7 +192,7 @@ main(int argc, char *argv[])
     // Initialize L3-Logging
 #if L3_ENABLED
     const char *logfile = "/tmp/l3.c-server-test.dat";
-    int e = l3_init(logfile);
+    int e = l3_log_init(L3_LOG_MMAP, logfile);
```

Comment:
Currently, I only need this performance benchmarking program to tap-into `l3_log_init()`, so it can be extended to other logging-schemes.

I am leaving the other call-sites which use `l3_init()` untouched at their default mode of mmap-logging. 
That change would needlessly introduce more code-diffs, for this kind of interface change that may not survive in future (tbd).